applications/GeoMechanicsApplication/tests/test_partially_saturated/test_saturated_below_phreatic_level_pw_triangle3N/ProjectParameters_stage1.json

```
@@ -0,0 +1,191 @@
+{
+  "problem_data": {
+    "problem_name"                     : "test_saturated_below_phreatic_level_pw_triangle3N_stage1",
+    "start_time"                       : -0.1,
+    "end_time"                         : 0.0,
+    "echo_level"                       : 1,
+    "parallel_type"                    : "OpenMP",
+    "number_of_threads"                : 1
+  },
+  "solver_settings": {
+    "solver_type"                      : "Pw",
+    "model_part_name"                  : "PorousDomain",
+    "domain_size"                      : 2,
+    "model_import_settings": {
+      "input_type"                     : "mdpa",
+      "input_filename"                 : "column"
+    },
+    "material_import_settings": {
+      "materials_filename"             : "../Common/MaterialParameters_stage1.json"
+    },
+    "time_stepping": {
+      "time_step"                      : 0.1,
+      "max_delta_time_factor"          : 1.0
+    },
+    "buffer_size"                      : 2,
+    "echo_level"                       : 1,
+    "clear_storage"                    : false,
+    "compute_reactions"                : true,
+    "reform_dofs_at_each_step"         : false,
+    "nodal_smoothing"                  : false,
+    "block_builder"                    : true,
+    "solution_type"                    : "Steady_State_Groundwater_Flow",
+    "scheme_type"                      : "Backward_Euler",
+    "reset_displacements"              : true,
+    "strategy_type"                    : "newton_raphson",
+    "convergence_criterion"            : "water_pressure_criterion",
+    "water_pressure_relative_tolerance": 0.0001,
+    "water_pressure_absolute_tolerance": 1E-09,
+    "displacement_relative_tolerance"  : 0.001,
+    "displacement_absolute_tolerance"  : 1E-09,
+    "residual_relative_tolerance"      : 0.0001,
+    "residual_absolute_tolerance"      : 1E-09,
+    "min_iterations"                   : 6,
+    "max_iterations"                   : 30,
+    "number_cycles"                    : 1,
+    "reduction_factor"                 : 1.0,
+    "increase_factor"                  : 1.0,
+    "desired_iterations"               : 4,
+    "max_radius_factor"                : 10.0,
+    "min_radius_factor"                : 0.1,
+    "calculate_reactions"              : true,
+    "max_line_search_iterations"       : 5,
+    "first_alpha_value"                : 0.5,
+    "second_alpha_value"               : 1.0,
+    "min_alpha"                        : 0.1,
+    "max_alpha"                        : 2.0,
+    "line_search_tolerance"            : 0.5,
+    "rotation_dofs"                    : true,
```

I don't think you want to have rotational DoFs in this analysis (it should be water pressure DoFs only, I think)...
```suggestion
    "rotation_dofs"                    : false,
```